src/time_spec.ts

```diff
@@ -2,7 +2,7 @@ import { describe, it } from 'mocha'
 import { expect } from 'chai'
 import { wrapPromiseWithTimeout } from './time'
 
-describe.only('wrapPromiseWithTimeout()', () => {
```

Comment:
Had accidentally added this in a previous PR, added the mocha options forbid-only so unit tests error if this is committed.